scripts/secrets/policies/node-ci.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -Eeuo pipefail
+
+VAULT_SECRETS_ROLE=${VAULT_SECRETS_ROLE:-ae-inventory}
+
+source ${DIR}/policies/ci.sh
+
+ROCKET_HOOK_URL=$(vault read -field=core-alerts secret/rocketchat/prod/hooks)
+
+dump_var "ROCKET_HOOK_URL"
```

Comment:
Based on this `dump_var` not defined in this file, and seeing that intended usage of this file seems to source it, it seems to me that `#!/bin/bash` shebang is misleading. Shall not `#!/bin/bash` shebang be deleted from files not meant to be executed? I also wonder if a conventional extension exists for these case.

Reply:
I was in doubt when adding them as well. I removed it in 79eb61b